### ISSUE TITLE
Captcha isn't working after docs instructions

### ISSUE THREAD

**Issue by hugofdsantos** [Azure/api-management-developer-portal#1791]
## Bug description

Captcha isn't working after following docs instructions.

## Reproduction steps

1. Enable captcha in config files
2. Publish website
3. Go to Signup page
4.  See **not found** error in browser developer tools network tab
5. 
![image](https://user-images.githubusercontent.com/20343825/171270611-2e1dc9c5-c041-4c5b-b881-6fb36850cd33.png)

6.
![image](https://user-images.githubusercontent.com/20343825/171270519-3349e9bc-5a75-484d-9431-a068bef85ebd.png)

## Expected behavior

Captcha should be displayed.

## Is your portal managed or self-hosted?

Self-hosted

## Release tag or commit SHA (if using self-hosted version)

release `2.14.0`, commit `47641f86ed21cfcb4414373af7f0fe5ee5882ddf`

## Environment

 - Operating system: [Windows 10]
 - Browser: [Brave, Firefox]

## Additional context

Looking into the code, I could see that the webpack pipeline isnt copying the hipObject.js file.
I got captcha working after pasting this following piece of code: `{ from: './js/HipObject.js', to: "assets/scripts/js" }`  in `webpack.publisher.js` `CopyWebpackPlugin` session.

**Comment by ygrik** [comment]
Hi @hugofdsantos 
Unfortunately, I can't reproduce your issue.
I get the latest code from the master and published website. All scripts loaded with 200
![image](https://user-images.githubusercontent.com/8660244/171501646-e061cbf9-286f-43e7-a056-7738a3eed860.png)

If it is possible, could you please share more details?

**Comment by hugofdsantos** [comment]
Hello @ygrik
Apparently, this was fixed in this commit: https://github.com/Azure/api-management-developer-portal/commit/0e0edd64406470368498eb1756e14bbc07885710#diff-1104240bfb29b2731c23e6bc153def822769d49a7a8c416f26f01222111d6fd4
By getting the latest code from master, I was able to get it working as well (I'm using the 2.14 version).
Maybe we can close this issue? It'll be solved when this commit gets released.
